<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>623</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="BHTab">
       <attribute name="title">
        <string>Basin Hopping</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="Show3D" name="widget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Energy (id)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listWidget">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushTakestepExplorer">
              <property name="text">
               <string>Takestep explorer</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnRun">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Start a short basinhopping run&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Run</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="NEBTab">
       <attribute name="title">
        <string>Connect</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="Show3D" name="oglPath" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="sliderFrame">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0,0">
            <item>
             <widget class="QListWidget" name="listMinima1">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="listMinima2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_2">
              <property name="leftMargin">
               <number>20</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>20</number>
              </property>
              <item row="1" column="0">
               <widget class="QPushButton" name="btnAlign">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Find best alignment between two structures&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>Align</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QPushButton" name="btnReconnect">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Start a fresh double ended connect run&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>Reconnect</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QPushButton" name="btnInvert">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Invert one of the structures&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>Invert</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QPushButton" name="btnNEB">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Start an NEB run (no alignment is done)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>NEB</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QPushButton" name="btnConnect">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Start a double ended connect run&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>Connect</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QPushButton" name="btn_close_all">
                <property name="text">
                 <string>Close Windows</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QPushButton" name="btnShowGraph">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Show the graph of minima and transition states&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>show graph</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QPushButton" name="btnDisconnectivity_graph">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Show the disconnectivity graph&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>disconnectivity graph</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QPushButton" name="btnconnect_in_optim">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Spawn an external OPTIM job&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="text">
                 <string>Connect in OPTIM</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QPushButton" name="btn_connect_all">
                <property name="text">
                 <string>Connect All</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="TSTab">
       <attribute name="title">
        <string>Transition States</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="Show3DWithSlider" name="oglTS" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <layout class="QVBoxLayout" name="verticalLayout_9" stretch="0">
          <item>
           <widget class="QListWidget" name="list_TS">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>839</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="actionConnect"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionDelete_Minimum"/>
    <addaction name="actionMerge_Minima"/>
    <addaction name="actionEditParams"/>
   </widget>
   <addaction name="menuSimulation"/>
   <addaction name="menuHelp"/>
   <addaction name="menuActions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear Database</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect to Database</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDelete_Minimum">
   <property name="text">
    <string>Delete Minimum</string>
   </property>
  </action>
  <action name="actionEditParams">
   <property name="text">
    <string>Edit default parameters</string>
   </property>
  </action>
  <action name="actionMerge_Minima">
   <property name="text">
    <string>Merge Minima</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Show3DWithSlider</class>
   <extends>QWidget</extends>
   <header>show3d.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Show3D</class>
   <extends>QWidget</extends>
   <header>pygmin/gui/show3d.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>listWidget</sender>
   <signal>currentItemChanged(QListWidgetItem*,QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>SelectMinimum()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>816</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRun</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>StartBasinHopping()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>816</x>
     <y>578</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAlign</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>AlignMinima()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>652</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnNEB</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ConnectMinima()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>652</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>526</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listMinima1</sender>
   <signal>currentItemChanged(QListWidgetItem*,QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>SelectMinimum1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>635</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>551</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listMinima2</sender>
   <signal>currentItemChanged(QListWidgetItem*,QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>SelectMinimum2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>635</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>536</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderFrame</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>showFrame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>579</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>537</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionConnect</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>connect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnInvert</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>Invert()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>796</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>687</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnConnect</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>doubleEndedConnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>652</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel">
     <x>685</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnShowGraph</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>show_graph()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>652</x>
     <y>544</y>
    </hint>
    <hint type="destinationlabel">
     <x>691</x>
     <y>476</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDelete_Minimum</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>delete_minimum()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDisconnectivity_graph</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>show_disconnectivity_graph()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>796</x>
     <y>577</y>
    </hint>
    <hint type="destinationlabel">
     <x>689</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnReconnect</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>doubleEndedReConnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>796</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel">
     <x>687</x>
     <y>402</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEditParams</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>edit_params()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnconnect_in_optim</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>connect_in_optim()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>796</x>
     <y>544</y>
    </hint>
    <hint type="destinationlabel">
     <x>833</x>
     <y>565</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMerge_Minima</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>merge_minima()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>list_TS</sender>
   <signal>currentItemChanged(QListWidgetItem*,QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_select_TS()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>745</x>
     <y>448</y>
    </hint>
    <hint type="destinationlabel">
     <x>835</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SelectMinimum()</slot>
  <slot>StartBasinHopping()</slot>
  <slot>NewSystem()</slot>
  <slot>SelectMinimum1()</slot>
  <slot>SelectMinimum2()</slot>
  <slot>AlignMinima()</slot>
  <slot>ConnectMinima()</slot>
  <slot>showFrame()</slot>
  <slot>showEnergies()</slot>
  <slot>save()</slot>
  <slot>load()</slot>
  <slot>tsSearch()</slot>
  <slot>selectTransition()</slot>
  <slot>showFrameTS()</slot>
  <slot>connect()</slot>
  <slot>Invert()</slot>
  <slot>LocalConnect()</slot>
  <slot>doubleEndedConnect()</slot>
  <slot>show_graph()</slot>
  <slot>delete_minimum()</slot>
  <slot>show_disconnectivity_graph()</slot>
  <slot>doubleEndedReConnect()</slot>
  <slot>edit_params()</slot>
  <slot>connect_in_optim()</slot>
  <slot>merge_minima()</slot>
  <slot>launch_connect_explorer()</slot>
  <slot>on_select_TS()</slot>
 </slots>
</ui>
